test/functional/rpc_signrawtransaction.py

```diff
@@ -4,6 +4,8 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test transaction signing using the signrawtransaction* RPCs."""
 
+from feature_cltv import CLTV_HEIGHT
+from feature_csv_activation import CSV_ACTIVATION_HEIGHT
```

Comment:
No strong opinion but I think moving these constants to `blocktools` would make some sense; as `COINBASE_MATURITY` is already there, and all three tests using them already import from there.

Reply:
Thanks, that seems like a good place to put them. Done.